docs/nx-cloud/intro/nx-cloud-workflows.md

```diff
@@ -0,0 +1,58 @@
+# Nx Cloud Workflows
+
+{% youtube
+src="https://www.youtube.com/embed/IQ5YyEYZw68?start=3371"
+title="Introducing Nx Cloud Workflows"
+width="100%" /%}
+
+## Powerful CI Capabilities Optimized for Nx monorepos
```

Comment:
I'm missing a bit the "Why" in this and next section. We describe what Nx Cloud workflow does etc, but not why you want to use it. We still need the latter (e.g. evolution of DTE, automatic scaling etc) but I think if we describe it from the "why beneficial" standpoint it resonates better

Things like
- "legacy" CI is low-level command execution on machines in the cloud
- they don't understand your workspace, just run commands
- Nx Cloud Workflows is task centric, you give it tasks to run and it knows how to split them into lots of fine-grained commands/targets based on your workspace structure
- cost savings due to automated DTE scaling, which before/right now you have to figure out almost on your own; like what number of agents makes most sense at your current monorepo situation to have good parallelism while still not wasting money by spinning up machines

something like this